.github/workflows/run_android_tests.yml

```
@@ -0,0 +1,60 @@
+name: android-tests
+
+on:
+  push:
+    branches:
+      - master
+      - main
+      - release/*
```

Please make sure to schedule the workflow on release and ciflow tags
```suggestion
      - release/*
    tags:
      # NOTE: Binary build pipelines should only get triggered on release candidate builds
      # Release candidate tags look like: v1.11.0-rc1
      - v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+
    - 'ciflow/trunk/*'
    - 'ciflow/android/*'
```